packages/gestalt/src/Borders.css

```diff
@@ -119,3 +119,7 @@ html[dir="rtl"] .borderLeft {
 .solid {
   border-style: solid;
 }
+
+.shadow {
+  box-shadow: 0 0 8px var(--g-colorTransparentGray100);
+}
```

Comment:
@themindfuldev looks like this is missing a newline at the end

![image](https://user-images.githubusercontent.com/127199/93628031-b7abf080-f99a-11ea-88ad-b9da036944c2.png)